source/language/classical.rst

```diff
@@ -79,6 +93,13 @@ an index set, for example ``i in {0,3}`` returns ``true`` if i equals 0 or 3 and
    // Susceptible to floating point casting errors
    e == float(d);
 
+.. note::
+
+   Angles are naturally defined on a ring, and so comparisons may not work
+   exactly how you might expect.  For example, ``2 * ang >= ang`` is not
+   necessarily true; if ``ang`` represents the angle :math:`3\pi/2`, then
+   ``2*ang == pi`` and ``pi < ang``.
```

Comment:
**question**: It might not be intuitive, but is it always well-defined? If so, it might be worth illustrating with concrete examples. If not, I think we should explicitly state it is not well-defined.

Reply:
I am open to forbidding the ordering comparisons of `angle` (i.e. disallowing `<`, `<=`, `>` and `>=`, but permitting `==` and `!=`).  I think we need more input from more people on that, if so.

If that's not what we decide, do you have a suggestion to make it clearer here (I'm totally open to that)?  It is always well defined (the semantics are very nearly the same as same operation between two `uint` with the same bit patterns), and I'm hoping the rest of the text of this PR (after review) will make that clear.

I was hoping the example in the second sentence would illustrate the "it's not always intuitive" view, but I'm happy to add to it.

Reply:
Yeah, for sure it's the same thing with two `uint`.  I think with `uint`, though, people who want to use the comparison operators are generally working with numbers that definitely won't overflow.  With `angle` here, the overflow will get hit all the time.

I've added an extra little bit in 986fade.